draft-sharabayko-mops-srt.md

```diff
@@ -740,8 +888,17 @@ Too-Late Packet Drop ({{too-late-packet-drop}}) mechanisms are used in this mode
 Buffer mode is negotiated during the Handshake by setting the STREAM flag
 of the handshake Extension Message Flags to 1.
 
-In this mode the consecutive packets form one consecutive stream that can be read with
-the portions of any size.
+In this mode consecutive packets form one continuous stream that can be read, with 
+portions of any size.
+
+
+## Stream Multiplexing
+
+Multiple SRT sockets may share one UDP socket, and the packets received on this
+UDP socket will be correctly dispatched to the SRT socket to which they are currently 
+destined. The parties exchange their SRT Socket IDs during the handshake. These IDs are 
+then used in the Destination Socket ID field of every control and data packet.
+
```

Comment:
This paragraph was moved into the "SRT Data Transmission and Control" section above (line 831).
So this is a repetition.

Reply:
Will delete it.